.github/workflows/staging-build-tar.yml

```diff
@@ -459,3 +459,50 @@ jobs:
           working-directory: kibana/plugins/anomaly-detection-kibana-plugin
           command: yarn cy:run-with-security --config baseurl=http://localhost:5601
     
+  Test-Perf-Analyzer-No-Security:
+    runs-on: ubuntu-18.04
+    name: Test-Perf-Analyzer-No-Security
+    strategy:
+      matrix:
+        java: [12]
```

Comment:
Please add a comment in the yml on java 12, this should be a temp measure and we should eventually switch to 14 just like other plugins.